src/components/Flipper/Flipper.tsx

```diff
@@ -371,8 +417,7 @@ function Flipper() {
                                 <span style={{ cursor: "pointer" }}> <SettingsIcon /></span>
                             </Form.Group>
                             {
-                                hasPremium ?
-                                    <span>The flipper is stuck <Tooltip type="hover" content={<HelpIcon />} tooltipContent={<span>We get new auctions every 60 sec. from Hypixel. So you may have to wait a bit for new ones to be found.</span>} /></span> : ""
+                                hasPremium ? <span>Expected new flips in: {lastFlipFetchTimeSeconds && !lastFlipFetchTimeLoading ? <Countdown date={getCountdownDateObject()} onComplete={onCountdownComplete} /> : "..."}</span> : ""
```

Comment:
```suggestion
                                hasPremium ? <span>Next update: {lastFlipFetchTimeSeconds && !lastFlipFetchTimeLoading ? <Countdown date={getCountdownDateObject()} onComplete={onCountdownComplete} /> : "..."}</span> : ""
```
Also could the countdown be reduced to only seconds since it never is more than 60 seconds